<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8CTCSTI" failures="1" tests="2" name="Test" time="50.374" errors="0" timestamp="2022-03-17T22:13:59 IST">
  <testcase classname="com.mystore.testcases.LoginPageTest" name="loginTest" time="7.219"/>
  <testcase classname="com.mystore.testcases.LoginPageTest" name="loginTest" time="17.703">
    <failure type="java.lang.AssertionError" message="test case sucssesfully passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: test case sucssesfully passed expected [true] but found [false]
at com.mystore.testcases.LoginPageTest.loginTest(LoginPageTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test -->
